test/upload-test-images.sh

```diff
@@ -32,7 +33,7 @@ function upload_test_images() {
 
   # ko resolve is being used for the side-effect of publishing images,
   # so the resulting yaml produced is ignored.
-  ko resolve ${tag_option} -RBf "${image_dir}" > /dev/null
+  ko resolve --platform=${PLATFORM} ${tag_option} -RBf "${image_dir}" > /dev/null
```

Comment:
I'm not sure how safe this is. Would it be better if you only add the --platform flag only if the env variable PLATFORM has been set?

Reply:
I have to say, I don't see any risk in the initial code, as `linux/amd64` is stable value for default builds.

However the logic of another variable specification in the script is the same as in your suggestion, so I've updated the code. I think it's better to have similarity here.